csharp/Glacier2/Callback/Client/Client.csproj

```diff
@@ -12,7 +12,7 @@
     <SliceCompile Include="../slice/AlarmClock.ice" />
     <PackageReference Include="ZeroC.Ice" Version="$(IceVersion)" />
     <PackageReference Include="ZeroC.Glacier2" Version="$(IceVersion)" />
-    <PackageReference Include="ZeroC.Ice.Slice.Tools.CSharp" Version="$(IceVersion)" />
+    <PackageReference Include="ZeroC.Ice.Slice.Tools" Version="$(IceVersion)" />
```

Comment:
ZeroC.Ice.Slice.Tools should _not_ be a runtime dependency. Do you plan to fix this in a follow-up PR?

Reply:
I will